meson.build

```diff
@@ -107,3 +112,16 @@ if scdoc.found()
 		)
 	endforeach
 endif
+
+if get_option('zsh-completions')
+	zsh_files = files(
+		'completions/zsh/_makoctl',
+		'completions/zsh/_mako'
+	)
+	zsh_install_dir = datadir + '/zsh/site-functions'
+
+	install_data(zsh_files,
+			install_dir: zsh_install_dir,
```

Comment:
Extra indent here

Reply:
Should 3b6603f4dd0528d5fbf7a8c5844c26edb4d4ff52 fix this